docs/getting_started.rst

```diff
@@ -446,13 +446,13 @@ according to the chromosome info in the specified file.
     bigwig_fp = '../data/wgEncodeCaltechRnaSeqHuvecR1x75dTh1014IlnaPlusSignalRep2.bigWig'
 
     with open(chromsizes_fp) as f:
-        chromsizes = []
+        chromsizes_arr = []
```

Comment:
Is this related to the Bed track?

Reply:
No but it's an error in the documentation so I thought I'd straighten it out here.